apps/builder/app/builder/features/style-panel/controls/select/select-control.tsx

```diff
@@ -29,7 +28,6 @@ export const SelectControl = ({
 
   return (
     <Select
-      disabled={isAdvanced}
```

Comment:
if select has advanced value, do we show it in that select?

Reply:
Yep, the flag was never passed. If we decide to indicate, it can be done inside control.